src/test/resources/testbib/testjabref_292.bib

```diff
@@ -299,3 +299,4 @@ @siumed.edu
   url = {http://dx.doi.org/10.1016/j.jneumeth.2008.01.009}
 }
 
+@Comment{jabref-meta: databaseType:bibtex;}
```

Comment:
This is JabRef 3.x - do not change bib files created with old versions of JabRef